doc/changes/DM-29370.feature.rst

```diff
@@ -0,0 +1 @@
+Add 'amp' parameter to the formatters for the Exposure StorageClass, allowing single-amplifier subimage reads.
```

Comment:
Since you picked rst file extension you want to be using double back ticks around that `amp` argument I think (and maybe also StorageClass and Exposure.